mobot/mobot_client/air_drop_session.py

```diff
@@ -0,0 +1,197 @@
+        self.send_mob_to_customer(customer, source, amount_to_send_mob, True)
+        drop_session.bonus_coin_claimed = bonus_coins[random_index]
+        drop_session.save()
+        total_prize = Decimal(initial_coin_amount_mob + amount_in_mob)
+        self.messenger.log_and_send_message(
+            customer,
+            source,
+            ChatStrings.REFUND_SENT.format(amount=amount_to_send_mob.normalize(), total_prize=total_prize.normalize())
+        )
+        self.messenger.log_and_send_message(
+            customer, source, ChatStrings.PRIZE.format(prize=total_prize.normalize())
+        )
+        self.messenger.log_and_send_message(
+            customer,
+            source,
+            ChatStrings.AIRDROP_COMPLETED
+        )
+
+        if self.customer_has_store_preferences(customer):
+            self.messenger.log_and_send_message(
+                customer, source, ChatStrings.BYE
+            )
+            drop_session.state = SessionState.COMPLETED.value
+            drop_session.save()
+        else:
+            self.messenger.log_and_send_message(
+                customer, source, ChatStrings.NOTIFICATIONS_ASK
+            )
+            drop_session.state = SessionState.ALLOW_CONTACT_REQUESTED.value
+            drop_session.save()
+
+    def handle_drop_session_waiting_for_bonus_transaction(self, message, drop_session):
+        print("----------------WAITING FOR BONUS TRANSACTION------------------")
+        if message.text.lower() == "help":
+            self.messenger.log_and_send_message(
+                drop_session.customer,
+                message.source,
+                ChatStrings.AIRDROP_COMMANDS
+            )
+        elif message.text.lower() == "pay":
+            self.messenger.log_and_send_message(
+                drop_session.customer,
+                message.source,
+                ChatStrings.PAY_HELP
+            )
+        else:
+            self.messenger.log_and_send_message(
+                drop_session.customer,
+                message.source,
+                ChatStrings.AIRDROP_COMMANDS
+            )
+
+        amount_in_mob = mc.pmob2mob(drop_session.drop.initial_coin_amount_pmob)
+
+        value_in_currency = amount_in_mob * Decimal(
+            drop_session.drop.conversion_rate_mob_to_currency
+        )
+
+        self.messenger.log_and_send_message(
+            drop_session.customer,
+            message.source,
+            ChatStrings.AIRDROP_RESPONSE.format(
+                amount=amount_in_mob.normalize(),
+                symbol=drop_session.drop.currency_symbol,
+                value=value_in_currency
+            )
+        )
+
+    def handle_active_airdrop_drop_session(self, message, drop_session):
+        if drop_session.state == SessionState.READY_TO_RECEIVE_INITIAL.value:
+            self.handle_drop_session_ready_to_receive(message, drop_session)
+            return
+
+        if drop_session.state == SessionState.WAITING_FOR_BONUS_TRANSACTION.value:
```

Comment:
When we switch out to `models.IntegerChoices` we can drop all the `.value` calls.